apps/remix-ide/src/app/panels/file-panel.js

```diff
@@ -60,6 +61,7 @@ module.exports = class Filepanel extends ViewPlugin {
     var fileSystemExplorer = createProvider('localhost')
 
     self.remixdHandle = new RemixdHandle(fileSystemExplorer, self._deps.fileProviders.localhost, appManager)
+    self.gitHandle = new GitHandle()
```

Comment:
I would recommend to put GitHandle at the root of the project and not in the FilePanel property